sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -259,7 +263,7 @@ export class Receiver {
 /**
  * The SessionReceiver class can be used to receive messages from a session enabled Queue or
  * Subscription in a batch or by registering handlers.
- * Use the `getSessionReceiver` function on the QueueClient or SubscriptionClient to instantiate a
+ * Use the `createReceiver` function on the QueueClient or SubscriptionClient to instantiate a
```

Comment:
Do we have other old names in the comments that get turned into doc comments?

Reply:
All old name references are taken care of in #2008